schema_cache.rst

```diff
@@ -106,26 +106,52 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+Another option is to send a `database notification event <https://www.postgresql.org/docs/current/sql-notify.html>`_. To enable this feature, add this variable to the configuration file:
+
+.. code::
+
+  # postgrest.conf
+
+  db-channel-enabled = true
+
+This way, the schema cache can be reloaded from any PostgreSQL client executing the ``NOTIFY`` command on the default channel:
+
+.. code-block:: postgresql
+
+  NOTIFY pgrst
+
+  -- Works the same way as:
+
+  NOTIFY pgrst, 'reload schema'
+
+
+.. important::
+
+   The ``NOTIFY`` command is not compatible with connection poolers like PgBouncer in transaction pooling mode.
```

Comment:
Ah, it's not the NOTIFY that's not compatible, it's the LISTEN that we do internally.

(could need rewording)
```suggestion
   db-channel is not compatible with connection poolers like PgBouncer in transaction pooling mode. Set it to false if you're using PostgREST behind a connection pooler.
```

Reply:
I used the [SQL feature map](https://www.pgbouncer.org/features.html#sql-feature-map-for-pooling-modes) from PgBouncer as reference, it mentions both `LISTEN/NOTIFY` as incompatible. However, I like the way you worded it, it includes the whole feature there.